rust/agama-lib/src/software/settings.rs

```diff
@@ -0,0 +1,13 @@
+//! Representation of the software settings
+
+use crate::settings::{Settings, SettingValue};
```

Comment:
I am not sure why this use is needed. I do not see its usage. I only see `Settings` from agama_derive

Reply:
Because it is used in the derive macro. I have fixed it, so it is not needed anymore.